utils/rpcd-mod-attendedsysupgrade/files/attendedsysupgrade.defaults

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+[ -e /etc/config/attendedsysupgrade ] && exit 0
+
+touch /etc/config/attendedsysupgrade
+
+uci -q batch <<EOF
+set attendedsysupgrade.updateserver=updateserver
+set attendedsysupgrade.updateserver.url='https://betaupdate.libremesh.org'
```

Comment:
Ideally I would like to see an official LEDE server here, or a defunct example.org domain. Most people likely do not know about libremesh.